lighthouse-core/config/default-config.js

```diff
@@ -505,23 +515,25 @@ const defaultConfig = {
     'best-practices': {
       title: str_(UIStrings.bestPracticesCategoryTitle),
       auditRefs: [
-        {id: 'appcache-manifest', weight: 1},
-        {id: 'is-on-https', weight: 1},
-        {id: 'uses-http2', weight: 1},
-        {id: 'uses-passive-event-listeners', weight: 1},
-        {id: 'no-document-write', weight: 1},
-        {id: 'external-anchors-use-rel-noopener', weight: 1},
-        {id: 'geolocation-on-start', weight: 1},
-        {id: 'doctype', weight: 1},
-        {id: 'charset', weight: 1},
-        {id: 'no-vulnerable-libraries', weight: 1},
-        {id: 'js-libraries', weight: 0},
-        {id: 'notification-on-start', weight: 1},
-        {id: 'deprecations', weight: 1},
-        {id: 'password-inputs-can-be-pasted-into', weight: 1},
-        {id: 'errors-in-console', weight: 1},
-        {id: 'image-aspect-ratio', weight: 1},
-        {id: 'image-size-responsive', weight: 1},
+        // Trust & Safety
+        {id: 'is-on-https', weight: 1, group: 'trust-and-safety'},
+        {id: 'external-anchors-use-rel-noopener', weight: 1, group: 'trust-and-safety'},
```

Comment:
how would you feel about `best-practices-trust-and-safety`? right now basically every group prefixes the category or an abbreviation of it